sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/StreamingWriteTables.java

```diff
@@ -77,7 +86,9 @@ public WriteResult expand(PCollection<KV<TableDestination, TableRow>> input) {
     // different unique ids, this implementation relies on "checkpointing", which is
     // achieved as a side effect of having StreamingWriteFn immediately follow a GBK,
     // performed by Reshuffle.
-    tagged
+    TupleTag<Void> mainOutput = new TupleTag<Void>() {};
```

Comment:
Same usual comments about tuple tag anonymous classes capturing their surrounding context. Instead, give the tags a name, and setCoder on the tagged output collections until it works.